docs/formstack.rst

```diff
@@ -0,0 +1,16 @@
+Formstack
+===================
+
+********
+Overview
+********
+
+`Formstack <https://www.formstack.com/>`_ is a service that provides an advanced online form builder.
```

Comment:
This docs page is missing the authentication blurb and the quickstart, as seen [here](https://move-coop.github.io/parsons/html/stable/zoom.html) (or any other connector)